### DISCUSSION TITLE
2023-01-11 Weekly Meeting 90

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#330] [General]
Weekly Meeting 90 is happening on Wednesday, January 11, 2022 from 9:00 AM PST(17:00 UTC) - 10:00 AM PST (18:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Working Group Updates:
- [MSF delegates](https://github.com/omigroup/MSF-Delegates) (setting up the repo)
- [UX Research](https://github.com/omigroup/ux-research) 
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour* 
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- [M3 avatar interop](https://github.com/M3-org/avatar-interop#calendar)
- [OMI Website update](https://github.com/omigroup/website)
- any others?

### Discussions: 

- OMC 

- This is a time to build. What can we build?
  - Adding OMI to the Metaverse Street Jin demoed
  - M3 recently started a shop for wearables. This is a great example of things we could be doing and participating in. 
  - Let's build right now. 

### Future demo themes: 
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo (December 7!)
- `IPSME` and Me
- `Generative and AI tooling`
- Any other suggestions?

**Comment by madjin** [comment]
Quick rundown on M3 activities + grants

- Openvoxels:
  - We took a snapshot of all parcels / wearables last month
  - The Sandbox game assets are archived also, DCL WIP
  - New cycle begins tonight with new features to reward contributions with
  - Grants available to promote metaverse interop with these assets + more: https://app.dework.xyz/m3/openvoxels/view/board-l6rgqct4
- AI:
  - Folks have been making REALLY GOOD AI powered characters in 🦾-ai 
  - Grants available if you're a dev: https://app.dework.xyz/m3/ai-43567/view/open-tasks-lblry0t1
- Virtual Productions:
  - Planning weekly show to interview / talk to ya'll about your projects: https://app.dework.xyz/m3/m3tv-assets/overview
  - Playlist: https://www.youtube.com/watch?v=YBth2sSi0LM&list=PLp5K4ceh2pR3scCC5B1f_DTDzUlJzZ3IH
- Avatar Interop group:
  - 90% done making MetaFactory avatar wearables
  - Tons of progress collaborating on Anata VRM avatars
  - Going to be testing with Character Creator / boom-tools (avatar generation) soon
    - Both open source avatar creation tools
  - We're actively contributing feedback to various metaverse interop groups like OMI / MSF
  - Grants available to help with docs + templates for Webaverse MMO: https://hackmd.io/@XR/webaverse-mmo

**Comment by mrmetaverse** [comment]
# OMC
- The conference is delayed until late Summer 2023.

**Comment by OBWANDO** [comment]
http://www.o3de.org/
http://www.openmv.org/
https://www.openmv.org/community/foundational-interest-groups/digital-assets-fig/
https://www.openmv.org/community/foundational-interest-groups/virtual-worlds-and-simulations-fig/

Discord:  https://discord.gg/KyDsm8VEUS

https://www.youtube.com/@Open3DEngine

**Comment by mrmetaverse** [comment]
# MSF 
MSF notes: https://github.com/omigroup/MSF-Delegates/discussions/40

MSF's avatar interoperability exploratory group proposal (approved):
https://docs.google.com/document/d/1Cb21T3IaKYY2iSBUleM4MaUzQxswz6fBGuy7TxoZShA/edit

MSF's network exploratory group proposal (approved): 
https://docs.google.com/document/d/15sL02PpJ3obMKm6OwpQoqiJ-H4vXamuiYOQVq7Q6pvI/edit

**Reply by mrmetaverse** [reply]
Looking at defining standard default posing, terminology, animation linkage, metadata attachment points, and more. Now that the proposal has been approved, the group will begin working on it. 

They also mentioned the membership tier proposal. There have been a number of good questions from the community, regarding the fidelity and impact one can have as a free member versus a principal, versus a corporate* sponsor (top tier). The community is doing a good job of staying mindful about keeping it open.

**Reply by mrmetaverse** [reply]
The 3D asset interop group meets at 11AM PT today. 
They meet in zoom. This can be streamed to our discord today to help ensure others can participate.

**Comment by mrmetaverse** [comment]
# UX Research 

Next meeting on Jan 17

**Comment by mrmetaverse** [comment]
# Dev hour 
- going dormant 
- Most of the dev hour work has been focused on thirdroom so the chat is migrating that direction.

**Comment by mrmetaverse** [comment]
# gLTF
- made changes to omi-link defining omi-links relation to the omi-collider 
- merged omi-collider spec and omi-physics-body spec
- omi-on-point merged 
- all of those are in stage 1/2 now. 
- To get involved you can look at the extensions here: https://github.com/omigroup/gltf-extensions/

**Comment by mrmetaverse** [comment]
# Media 
- yearly cleaning of pw and socials etc. 
- need to ensure access

**Comment by mrmetaverse** [comment]
# Website
If you have content that you think would be great on the website, write 2 paragraphs, and a link to the blog post. 

![image](https://user-images.githubusercontent.com/63426722/211877440-c10f335d-adf3-49dc-b1fb-12cd24c3d334.png)

**Comment by mrmetaverse** [comment]
# Demos 

Parzival and Moon have been collaborating on a really cool AI tool. With open source "spells" that will do many different AI things. This allows you to essentially inject personalities into your avatar. 

![image](https://user-images.githubusercontent.com/63426722/211877680-13e8a850-5a3e-4db3-8207-19432f094bac.png)

Can get into sentiment
![image](https://user-images.githubusercontent.com/63426722/211877713-3ad98aa6-5d74-4189-babd-d281e7afcb63.png)


Using MagickML you can get really deep and complicated 
![image](https://user-images.githubusercontent.com/63426722/211877762-161963b5-9456-4784-bcc3-46083bb03e39.png)

facial expression changes

![image](https://user-images.githubusercontent.com/63426722/211877855-ae93c027-8839-4610-85e6-53ef24a6c468.png)
![image](https://user-images.githubusercontent.com/63426722/211877868-f2e66ac1-e040-4f2b-8538-d558a4560249.png)


You can use openAI models to decide the sentiment of messages. 

![image](https://user-images.githubusercontent.com/63426722/211877924-61f1a688-45ad-4282-b18b-7b04660d781e.png)

You can map them to the VRM model to change avatar expressions. 

You can also write lore, and more telling them how they should act. 

Exploring a gLTF extension for personalities! 

![image](https://user-images.githubusercontent.com/63426722/211878048-bdc168f5-c4c8-4005-8f71-7c397f104071.png)

It is already getting implemented in multiple places, so this seems to be a good fit for an OMI spec. 

antpb also worked on a cloudflare worker for this: 

![image](https://user-images.githubusercontent.com/63426722/211878362-e93a9a48-bb19-4ae2-bdbc-3b6d29a478b1.png)

Check out MagickML for more: https://github.com/Oneirocom/MagickML

**Reply by mrmetaverse** [reply]
If you are technical and are interested in grants for this please see: https://github.com/omigroup/omigroup/discussions/330#discussioncomment-4658842

**Comment by mrmetaverse** [comment]
# Croquet new wordpress plugin (upcoming)

You can embed these little worlds in a post. 

![image](https://user-images.githubusercontent.com/63426722/211879720-e79ece9c-e695-494b-952a-af652de11490.png)
![image](https://user-images.githubusercontent.com/63426722/211879762-72632163-9672-4a3a-8ddd-3cfa21c5c016.png)
![image](https://user-images.githubusercontent.com/63426722/211879794-d7d000aa-a32a-4379-9f2d-1b661240e58f.png)

Deployable in just 5 minutes:

**Comment by mrmetaverse** [comment]
# Free 3d objects 
(DYOR on licenses) 

- https://lowesopenbuilder.com/
- https://amazon-berkeley-objects.s3.amazonaws.com/index.html

**Comment by mrmetaverse** [comment]
# Proposed RFC OpenXR 
- https://github.com/o3de/sig-graphics-audio/issues/39

**Comment by mrmetaverse** [comment]
# Open 3D Foundation 
Join here: discord.gg/o3de

![image](https://user-images.githubusercontent.com/63426722/211881940-4f24722a-9f1b-47a9-aa17-c9c02a243209.png)

**Reply by mrmetaverse** [reply]
The linux foundation is a non profit. They want to work with other communities like OMI to collaborate and share information. Leveraging interest groups: 
![image](https://user-images.githubusercontent.com/63426722/211882084-9ae09968-37ee-4795-948d-369b1798ef8b.png)

**Comment by mrmetaverse** [comment]
# Call for assistance 

- Looking for a volunteer to lead the open collective reporting. 
- we will hold an election to vote you in
